searchcore/src/vespa/searchcore/proton/matching/matcher.cpp

```diff
@@ -281,16 +280,28 @@ Matcher::match(const SearchRequest &request,
         reply = std::move(result->_reply);
         SearchReply::Coverage & coverage = reply->coverage;
         if (wasLimited) {
+            LOG(debug, "was limited, degraded from match phase");
             coverage.degradeMatchPhase();
         }
         if (my_stats.softDoomed()) {
+            LOG(debug, "soft doomed, degraded from timeout");
             coverage.degradeTimeout();
         }
-        coverage.setActive(metaStore.getNumActiveLids());
+        uint32_t numUsedLids = metaStore.getNumUsedLids();
```

Comment:
numUsedLids is too low, since the space estimate from the limiter counts unused lids as well.